mytimer/functions.py

```diff
@@ -301,14 +326,24 @@ def countdown_timer(
     :type tone: str
     :param alarm_repeat: alarm repeat
     :type alarm_repeat: int
+    :param v_shift: vertical shift
+    :type v_shift: int
+    :param h_shift: horizontal shift
+    :type h_shift: int
     :return: None
     """
+    prepend = ' ' * h_shift
     while True:
         start = time.perf_counter()
         clear_screen()
-        print('\n' * 5)
-        tprint('\t\t\t\t  %d : %d : %d ' %
-               (hour, minute, second), font=face)
+        print('\n' * v_shift, end='')
```

Comment:
Alternative :

```python
print(" " * h_shift, end='')
tprint(TIME_PRINT_TEMPLATE.format(timer_hour,timer_minute,timer_second), font=face, sep="\n" + " " * h_shift)
```

⚠️ We need to update the minimum version of `art` to `5.3`.